cmd/bugzilla-mass-cloner/README.md

```diff
@@ -0,0 +1,19 @@
+# Bugzilla Mass Cloner
+
+This tool can be used to mass clone bugs from a target release to a desired one.
+
+````sh
+Usage of ./bugzilla-mass-cloner:
+  -bugzilla-api-key-path string
+    	Path to the file containing the Bugzilla API key.
+  -bugzilla-endpoint string
+    	Bugzilla's API endpoint.
+  -bugzilla-github-external-tracker-id uint
+    	The ext_type_id for GitHub external bugs, optional.
```

Comment:
What is `ext_type_id` for GitHub external bugs?

Reply:
Oh, these options are from https://github.com/kubernetes/test-infra/blob/master/prow/flagutil/bugzilla.go. I am not sure what is this for, to be honest.